scripts/cosmos-wait

```diff
@@ -3,7 +3,7 @@
 echo "Waiting for CosmosDB"
 
 until wget --no-check-certificate --timeout=5 --tries=1 -O /dev/null -o /dev/null https://ffc-sfd-data-cosmos-db:8081/_explorer/index.html; do
-    sleep 5;
+    sleep 6;
```

Comment:
Had to increase the wait time for the Cosmos emulator to start up because I kept coming into this error:
```
Error: Failed to create databases: Failed to create files database: Timeout Error! Request took more than 60000 ms
ffc-sfd-data-development  |     at initCosmos (/home/node/app/cosmos/init.js:23:11)
ffc-sfd-data-development  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
ffc-sfd-data-development  |     at async init (/home/node/app/index.js:22:3)
```
Increasing the wait time by a second resolved this issue.